src/main/java/com/softserve/teamproject/entity/Group.java

```diff
@@ -182,25 +187,18 @@ public int hashCode() {
     return Objects.hash(name);
   }
 
-    @Override
-    public String toString() {
-      return "Group{"
-          + "id=" + id
-          + ", name='" + name + '\''
-          + ", teachers=" + teachers
-          + ", location=" + location.getName()
-          + ", startDate=" + startDate
-          + ", finishDate=" + finishDate
-          + ", status=" + status
-          + ", specialization=" + specialization
-          + '}';
-    }
-
-  public BudgetOwner getBudgetOwner() {
-    return budgetOwner;
-  }
-
-  public void setBudgetOwner(BudgetOwner budgetOwner) {
-    this.budgetOwner = budgetOwner;
+  @Override
+  public String toString() {
+    return "Group{"
+        + "id=" + id
+        + ", name='" + name + '\''
+        + ", teachers=" + teachers
+        + ", location=" + location.getName()
+        + ", startDate=" + startDate
+        + ", finishDate=" + finishDate
+        + ", status=" + status
+        + ", specialization=" + specialization
+        + ", budgetOwner=" + budgetOwner
```

Comment:
I feel like budgetOwner.getName() would be better, same for status and specialization